src/Spryker/Zed/EventBehavior/Persistence/Propel/Behavior/EventBehavior.php

```diff
@@ -493,6 +504,74 @@ protected function isEventColumnsModified()
         ";
     }
 
+    /**
+     * @return string
+     */
+    protected function addGetOriginalValuesMethod()
+    {
+        $tableName = $this->getTable()->getName();
+
+        $originalValueColumns = [];
+        $eventColumns = $this->getParameters();
+        foreach ($eventColumns as $eventColumn) {
+            if ($eventColumn['column'] === '*') {
```

Comment:
This code is really hard to read and understand, I suggest you split nested parts (`if`) into small methods with good naming

Reply:
Done, I hope it's better now.